### PR TITLE
[Doc] Correct the model cache encryption scope.

### PR DESCRIPTION
**Pull request by peterchen-intel**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/bindings/c/include/openvino/c/ov_property.h [hunk 1]
```diff
@@ -109,7 +109,7 @@ ov_property_key_cache_mode;
 
 /**
  * @brief Write-only property<ov_encryption_callbacks*> to set encryption/decryption function for model cache.
- * If ov_property_key_cache_encryption_callbacks is set, model cache will be encrypted/decrypted when saving/loading
+ * If ov_property_key_cache_encryption_callbacks is set, model topology in cache will be encrypted/decrypted when saving/loading
  * model cache. ov_property_key_cache_encryption_callbacks is enabled in ov_core_compile_model_* only
```
Comment (peterchen-intel) [inline src/bindings/c/include/openvino/c/ov_property.h:113]:
```suggestion
 * If ov_property_key_cache_encryption_callbacks is set, model topology in cache will be encrypted/decrypted when
 * saving/loading model cache. This property is enabled in ov_core_compile_model_* only
```

File: src/bindings/c/include/openvino/c/ov_property.h [hunk 2]
```diff
@@ -109,8 +109,8 @@ ov_property_key_cache_mode;
 
 /**
  * @brief Write-only property<ov_encryption_callbacks*> to set encryption/decryption function for model cache.
- * If ov_property_key_cache_encryption_callbacks is set, model cache will be encrypted/decrypted when saving/loading
- * model cache. ov_property_key_cache_encryption_callbacks is enabled in ov_core_compile_model_* only
+ * If ov_property_key_cache_encryption_callbacks is set, model topology in cache will be encrypted/decrypted when
+ * saving/loading model cache. This property is enabled in ov_core_compile_model_* only
```
Comment (ilya-lavrenov) [inline src/bindings/c/include/openvino/c/ov_property.h:113]: can we also update this document https://docs.openvino.ai/2024/openvino-workflow/running-inference/optimize-inference/optimizing-latency/model-caching-overview.html to reflect how to use this new feature?

**Review by xufang-lisa — APPROVED** [review]

File: src/bindings/c/include/openvino/c/ov_property.h (hunk 2 above)
Comment (xufang-lisa) [reply src/bindings/c/include/openvino/c/ov_property.h:113]: https://github.com/openvinotoolkit/openvino/pull/26432

**Comment by xufang-lisa** [comment]
replaced by https://github.com/openvinotoolkit/openvino/pull/26432

**Pull request closed without merging**